.eslintignore

```diff
@@ -0,0 +1 @@
+**/node_modules/**
```

Comment:
Shouldn't we also add submodule paths here?

Reply:
Added todo and exclude packages from linting